script/cadence/contracts/FlowColdStorageProxy.cdc

```diff
@@ -89,7 +89,7 @@ pub contract FlowColdStorageProxy {
         // "FLOW-V0.0-user" followed by a padding of null bytes.
         //
         // The resulting data is then hashed with SHA3-256 before being signed.
-        pub fun transfer(receiver: Address, amount: UFix64, nonce: Int64, sig: [UInt8]) {
+        access(all) fun transfer(receiver: Address, amount: UFix64, nonce: Int64, sig: [UInt8]) {
```

Comment:
I would normally think that this method should be entitled. But it looks like a signature signed by the Vault's assigned key is required to execute this transfer where the signed message corresponds to the receiver, amounts and nonce provided. In essence access control is gated here by signature rather than entitlement - is that right?

Reply:
Yeah, this is in keeping with the security scheme which is embedded into Rosetta which does everything with signatures and keys, a base level of consistency across all chains?